src/github-handler/fork-handler.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Logger, Octokit} from '../types';
+
+interface ForkData {
```

Comment:
We could probably generalize this to be `Repository` with `owner` and `repo` fields. A general repository type could be used for upstream and origin elsewhere.

Reply:
This specific interface is for returning 2 values from the fork function. There's not another instance in the code where just the owner and repo are passed/returned. If I move this to the general src/types then it will not be used by anything.

Reply:
My thought is that maybe `GitHubContext` and/or `GitHubContextParams` could specify `upstream: Repository` rather than `upstreamOwner`/`upstreamRepo`

Reply:
This is a non-blocker on this PR, but is something to consider later.